lib/ansible/modules/unarchive.py

```diff
@@ -500,7 +500,7 @@ def is_unarchived(self):
                 continue
 
             # Check first and seventh field in order to skip header/footer
-            if len(pcs[0]) != 7 and len(pcs[0]) != 10:
+            if len(pcs[0]) != 7 and len(pcs[0]) != 8 and len(pcs[0]) != 10:
```

Comment:
```
if len(pcs[0]) not in (7,8,10):
```

Reply:
@bcoca suggestion has been committed.

@Akasurde Changelog fragment added. This is already tested as part of the unarchive test cases.

Reply:
@GodsVictory If this case were already tested that test would have been failing. Tests were passing before this change, so this is not already tested.

Reply:
@flowerysong good point... Unfortunately this would require a solaris host with a specific version of zipinfo installed to test properly.  Apparently this issue has been mentioned before: https://github.com/ansible/ansible/issues/)35374#issuecomment-423078854